run.sh

```diff
@@ -9,6 +9,7 @@ if [[ $SUMO_ACCESS_KEY_FILE ]]; then
 fi
 
 SUMO_GENERATE_USER_PROPERTIES=${SUMO_GENERATE_USER_PROPERTIES:=true}
+SUMO_GENERATE_COLLECTOR_PROPERTIES=${SUMO_GENERATE_COLLECTOR_PROPERTIES:=true}
```

Comment:
Can you please mention about this in the `README` file as well? Similar to `SUMO_GENERATE_USER_PROPERTIES`.

Reply:
Done!